UefiPayloadPkg/Library/FdtParserLib/FdtParserLib.c

```diff
@@ -282,6 +288,15 @@ ParseReservedMemory (
           PlatformAcpiTable->Header.Revision = UNIVERSAL_PAYLOAD_ACPI_TABLE_REVISION;
           PlatformAcpiTable->Header.Length   = sizeof (UNIVERSAL_PAYLOAD_ACPI_TABLE);
         }
+      } else if (AsciiStrnCmp (TempStr, "smbios", AsciiStrLen ("smbios")) == 0) {
+        DEBUG ((DEBUG_INFO, " build smbios, NumberOfBytes:%x", NumberOfBytes));
+        BuildMemoryAllocationHob (StartAddress, NumberOfBytes, EfiReservedMemoryType);

```

Comment:
Just for optimization, we only need EfiBootServicesData here which can be re-used by OS later as the Smbios tables that bootloader provided here will be copied to EfiRuntimeServicesData buffer by SmBiosDxe driver.

Reply:
Makes sense. Done.

Reply:
resolved